scripts/ci-integration.sh

```diff
@@ -67,12 +68,33 @@ delete_bootstrap() {
    kind delete cluster --name "${BOOTSTRAP_CLUSTER_NAME}"
 }
 
-verify_cluster_api() {
-   echo "not implemented"
-   #TODO: verify cluster-api CRD on bootstrap cluster
+wait_pod_running() {
+   retry=30
+   INTERVAL=6
```

Comment:
Any reason these vars are in different cases?

Reply:
good catch, no strong reason, thanks